docs/en/reference/sql/ddl/CREATE_FUNCTION.md

```diff
@@ -43,4 +43,14 @@ CREATE FUNCTION cut2(x string) RETURNS string OPTIONS (FILE = 'libtest_udf.so');
 Create a function whose input parameter type is two ints, the return type is int, and the dynamic library file is libtest_udf.so
 ```sql
 CREATE FUNCTION add_one(x int, y int) RETURNS INT OPTIONS (FILE = 'libtest_udf.so');
+```
+
+Create an aggregate function whose input parameter type is bigint, the return type is bigint, and the dynamic library file is libtest_udf.so
+```sql
+CREATE AGGREGATE FUNCTION special_sum(x BIGINT) RETURNS BIGINT OPTIONS (FILE = 'libtest_udf.so');
+```
+
+Create an aggregate function whose input parameter type is bigint and nullable, the return type is bigint and nullable, and the dynamic library file is libtest_udf.so
+```sql
+CREATE AGGREGATE FUNCTION count_null(x BIGINT) RETURNS BIGINT OPTIONS (FILE = 'libtest_udf.so', ARG_NULLABLE=true, RETURN_NULLABLE=true);
```

Comment:
Maybe we can deduce the nullable info from the external function define, so the two nullable flags can omited.

Reply:
there is the method to get the type of argument from dynamic lib ?

Reply:
possible in cpp with template deduction. Just further enhancements